schema/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Config",
-  "description": "The config block controls a device's intended behavior. Read more: <https://github.com/faucetsdn/udmi/blob/master/docs/messages/config.md>",
+  "description": "The config block controls a device's intended behavior. [Read more](../../docs/messages/config.md)",
```

Comment:
not this file, but the "schema/README.md" file is empty?

Reply:
Removed - didn't intend to commit